tests/acl/test_acl_outer_vlan.py

```diff
@@ -284,13 +284,16 @@ def get_acl_counter(duthost, table_name, rule_name, timeout=ACL_COUNTERS_UPDATE_
     Returns:
         Acl counter value for packets
     """
-    cmd = "redis-cli -n 2 hget 'COUNTERS:{}:{}' Packets"
     # Wait for orchagent to update the ACL counters
     time.sleep(timeout)
-    result = duthost.shell(cmd.format(table_name, rule_name))['stdout']
-    if result == "":
-        pytest.fail("Failed to retrieve acl counter for {}|{}".format(table_name, rule_name))
-    return int(result)
+    result = duthost.show_and_parse('aclshow')
+
+    if len(result) == 0:
+        return 0
```

Comment:
Is the "0" expected behavior? If not, could we some add debug/error print here?

Reply:
@StormLiangMS If there is no rule for acl, the output looks like below,  result is empty.
We return 0 here. The log file includes all the output of shell command.
```
admin@str-dx010-acs-5:~$ aclshow -a
RULE NAME    TABLE NAME    PRIO    PACKETS COUNT    BYTES COUNT
-----------  ------------  ------  ---------------  -------------
```

Reply:
@StormLiangMS Rethinking about your comment, you are correct. get_acl_counter is called after setup acl rules, so if can't find any expected rule, fail the test case.